utils/webservice.go

```diff
@@ -198,7 +198,7 @@ func (hcs *HTTPConcertoservice) Get(path string) ([]byte, int, error) {
 }
 
 // GetFile sends GET request to Concerto API and receives a file
-func (hcs *HTTPConcertoservice) GetFile(path string, directoryPath string) (string, int, error) {
+func (hcs *HTTPConcertoservice) GetFile(path string, directoryPath string, fileName string) (string, int, error) {
```

Comment:
Initially, the path for downloaded file was an output. Finally i implemented it focus on receiving it as an input